test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -760,3 +760,28 @@
 When(/^I click the review item column in the tag table$/) do
   page.all('td.f1').first.click
 end
+
+Then(/^I should see a node element "([^"]*)" with a selected highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.selected'
+end
+
+Then(/^I should see element "([^"]*)" with a yellow highlight$/) do |id|
+  el = find('div.layer-data').first('g[class*=" ' + id + '"]')
+  el[:class].include? '.edited.unsaved'
+end
+
+And(/^I should see an element with class "([^"]*)" on the map$/) do |id|
+  el = find('div.layer-data')
+  el[:class].include? id
```

Comment:
Does this "include" actually find children of div.layer-data with the class?
![screenshot from 2016-09-21 12 49 13](https://cloud.githubusercontent.com/assets/2600932/18720439/d30eb53e-7ff9-11e6-9909-31a85bfda2e3.png)